twentytwentyone-blocks/assets/css/style-shared.css

```diff
@@ -11,13 +11,31 @@ body {
 h1,
 .has-huge-font-size,
 .has-gigantic-font-size {
-	font-weight: var(--wp--custom--font-weight-light);
+	font-weight: var(--wp--preset--font-weight-light);
 }
 
 h2, h3, h4, h5, h6 {
 	font-weight: normal;
 }
 
+b,
+strong {
+	font-weight: var(--wp--preset--font-weight--bold);
+}
+
+dfn,
+cite,
+em,
+i {
+	font-style: italic;
```

Comment:
I feel like these should be default styles provided by core

Reply:
https://github.com/WordPress/gutenberg/issues/27269